gameday/dto.go

```diff
@@ -41,3 +46,54 @@ type LookupTeamDTO struct {
 	Label string `json:"label"`
 	Value string `json:"value"`
 }
+
+// GamedayDTO the data transfer object for
+// creating a new gameday
+type GamedayDTO struct {
+	Name        string
+	Team        LookupTeamDTO
+	ScheduledAt ScheduledAtTime `json:"schedule_at"`
+}
+
+// Validate check if the DTO has the required values
+func (g GamedayDTO) Validate() error {
+	if g.Name == "" {
+		return errors.New("failed: missing required field name")
+	}
+	if g.Team.Value == "" {
+		return errors.New("failed: missing required field team ID")
+	}
+	if g.ScheduledAt.String() == "" {
+		return errors.New("failed: missing required field scheduled_at")
+	}
+	return nil
+}
+
+// ScheduledAtTime time for `scheduled_at`
+type ScheduledAtTime time.Time
+
+const timeLayout = "2006-01-02 15:04:05"
```

Comment:
Shouldn't we apply on format TZ? Or do we assume silently UTC?

Reply:
silently UTC I was thinking